### DISCUSSION TITLE
Prometheus non operator installation on ocp without cluster admin privilege

### DISCUSSION THREAD

**Discussion by phaneesh11** [prometheus/prometheus#15566] [Q&A]
### What did you do?

I am following below steps on ocp4.16.where I have access to project and I am not a cluster admin, I have attached values.yaml
[Uploading prom-values.yaml.yaml.zip…](https://github.com/artifacthub/hub/issues/4169)

helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
helm install my-prometheus prometheus-community/prometheus --version 26.0.0

Getting below error
[root@ad cos]# helm install obs-prometheus prometheus-community/prometheus --version 26.0.0 --values values.yaml
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /root/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /root/.kube/config
Error: INSTALLATION FAILED: Unable to continue with install: could not get information about the resource ClusterRoleBinding "obs-prometheus-kube-state-metrics" in namespace "": clusterrolebindings.rbac.authorization.k8s.io "obs-prometheus-kube-state-metrics" is forbidden: User "rtmsadmin" cannot get resource "clusterrolebindings" in API group "rbac.authorization.k8s.io" at the cluster scope

### What did you expect to see?

Installation succesful without any issues as project admin

### What did you see instead? Under which circumstances?

Getting below error
[root@ad cos]# helm install obs-prometheus prometheus-community/prometheus --version 26.0.0 --values values.yaml
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /root/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /root/.kube/config
Error: INSTALLATION FAILED: Unable to continue with install: could not get information about the resource ClusterRoleBinding "obs-prometheus-kube-state-metrics" in namespace "": clusterrolebindings.rbac.authorization.k8s.io "obs-prometheus-kube-state-metrics" is forbidden: User "rtmsadmin" cannot get resource "clusterrolebindings" in API group "rbac.authorization.k8s.io" at the cluster scope
[prom-values.yaml.yaml.zip](https://github.com/user-attachments/files/18010709/prom-values.yaml.yaml.zip)

**Comment by beorn7** [comment]
This doesn't seem to be a bug in Prometheus. It looks more like a config problem, maybe an issue with the Helm chart. I'll convert this into a discussion, but I would also recommend to look for help via Helm or [Prometheus](https://prometheus.io/community/) community channels.
